docs/src/Typeahead.doc.js

```diff
@@ -353,4 +360,65 @@ function Example(props) {
   />
 );
 
+card(
+  <Example
+    id="zIndex"
+    name="With ZIndex"
+    description={`
+    Modals within Layer components with zIndexs will overlay any children component. Use zIndex prop in Typeahead to bring the Typeahead list to the top of the zIndex stack.`}
+    defaultCode={`
+function Example(props) {
+  const [open, setOpen] = React.useState(false);
+  const HEADER_ZINDEX = new FixedZIndex(10);
+  const MODAL_ZINDEX = new CompositeZIndex([HEADER_ZINDEX]);
+  const TYPEAHEAD_ZINDEX = new CompositeZIndex([MODAL_ZINDEX]);
```

Comment:
Should the array passed in here also include HEADER_ZINDEX? Though trivial here, if that array is intended to include all z-indices that TYPEAHEAD_ZINDEX should be above, it seems like we should include the header one as well.

Reply:
In this case, it's not necessary as compositeZIndex is adding +1 under the hood, and we have them linked

HEADER_ZINDEX > MODAL_ZINDEX > TYPEAHEAD_ZINDEX

My assumption is that there could be cases where we want to get overlay the highest of different Zindexes and provide that flexibility to the engineer.

I don't think it's the case here.